app/views/api/v2/app_visitor_configs/_show.json.jbuilder

```diff
@@ -0,0 +1,10 @@
+json.splits active_splits do |split|
+  json.name split.name
+  json.weights split.registry
+  json.feature_gate split.feature_gate?
+end
+
+json.visitor do
+  json.id visitor_id
+  json.assignments assignments, :split_name, :variant
```

Comment:
and we dropped the context and unsynced attributes here since they're related to syncing unsynced assignments to analyticsland on fetch of the visitor assignments which we're not going to be doing anymore in clients